### DISCUSSION TITLE
Can I use the example codes for the lecture ? ( Sorry for bothering )

### DISCUSSION THREAD

**Discussion by wo-o** [argoproj/argo-rollouts#2871] [General]
Hi, team.
I am sorry but can I use the example codes for the lecture?

I know it is on Apache 2.0, so commercial use is not restricted.
But I just want to make double check also for the examples.
(https://github.com/argoproj/argo-rollouts/tree/master/examples)

Sorry again for the bothering, but please kindly let me know.

**Comment by kostis-codefresh** [comment]
Yes you can use the examples in the lecture.
